setup.py

```diff
@@ -33,7 +33,6 @@
               'Intended Audience :: Education',
               'License :: OSI Approved :: BSD License',
               'Operating System :: OS Independent',
-              'Programming Language :: Python :: 2.7',
               'Programming Language :: Python :: 3',
               'Programming Language :: Python :: 3.5',
               'Programming Language :: Python :: 3.6',
```

Comment:
We also need to make it more explicit that the next version of joblib cannot be installed on Python 2 anymore:

```python
                    python_requires=">=3.6",
```

Reply:
And we can also probably drop support for Python 3.5 while we are at it.